### PR TITLE
Fix: Remove v6 adapted and fix scaler issues

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

File: tests/test_ssim_sonata.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def test_sim_quick_scx_sonata(input_type):
     mainsim_voltage = sim.get_mainsim_voltage_trace(cell_id)
     voltage_diff = voltage - mainsim_voltage
     rms_error = np.sqrt(np.mean(voltage_diff ** 2))
-    assert rms_error < 1e-4
+    assert rms_error < 2e-2
```
Comment (anilbey) [inline tests/test_ssim_sonata.py:43]: Is this rms_error change related to this PR or to a more general update on NEURON?
The tests still pass when rms_error stays the same (at least on an ubuntu 22.04).

Comment (arnaudon) [reply tests/test_ssim_sonata.py:43]: ah this was on my machine, if it passes without, I can revert this change, let me try here

Comment (arnaudon) [inline tests/test_ssim_sonata.py:43]:
```suggestion
    assert rms_error < 1e-4
```

**Review by anilbey — APPROVED** [review]
Looks great, thanks!

**Pull request merged**